ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/YoutubeQualityScriptHandler.swift

```diff
@@ -113,4 +124,18 @@ class YoutubeQualityScriptHandler: NSObject, TabContentScript, TabObserver {
   func tabWillBeDestroyed(_ tab: some TabState) {
     tab.removeObserver(self)
   }
+
+  private static func handleConnectionStatusChanged(
```

Comment:
We wouldn't want to do this unless the user is using "only on wifi" setting

Reply:
Also we dont seem to give the user any choice, is `medium` the default?

Reply:
The default I think is "auto", but we already do that technically. 
If I set it to "auto" or "", youtube will still play in the highest quality setting as it was already set, **until you refresh** or until the page discard happens.

I have changed it back from `medium` to `auto`. When the page is restored, it will show `auto` or when the page is reloaded by YT, it will show `auto`